doc-content/enterprise-only/project-data/preview-editor-create-test-scenario-mortgages-example-proc.adoc

```diff
@@ -0,0 +1,76 @@
+[id='preview-editor-create-test-scenario-mortgages-example-proc']
+= Creating test scenario using the sample Mortgages project
+
+In this chapter, as an example for better understanding, you will create and execute a test scenario from the sample *Mortgages* project shipped with {CENTRAL} using the *Test Scenario (Preview)* editor. The test scenario that you will create is based on the *Pricing loans* guided decision table from the *Mortgages* project.
```

Comment:
Avoid using "will". Suggestion:
The test scenario example in this chapter is based on the *Pricing loans* guided decision table from the *Mortgages* project.

Reply:
Changed as suggested.